docs/architecture.md

```diff
@@ -1,37 +1,53 @@
 # Architecture
 
-**Note:** As with any young project, things change rapidly. Thus all the details in this architecture documentation may not be always up-to-date, but the high level concepts and patterns should still apply.
+**Note:** As k0s is a new and dynamic project, the product architecture may
+occasionally outpace the documentation. The high level concepts and patterns, however, should always apply.
 
 ## Packaging
 
-k0s is packaged as single, self-extracting binary which embeds Kubernetes binaries. This has many benefits:
-- Everything can be, and is, statically compiled
-- No OS level deps
-- No RPMs, dep's, snaps or any other OS specific packaging needed. Single "package" for all OSes
-- We can fully control the versions of each and every dependency
+The k0s package is a single, self-extracting binary that embeds Kubernetes
```

Comment:
Remove the manual line changes (new line characters) from the middle of the paragraphs (this is general comment for all changes). Otherwise it becomes difficult to modify the text later on and you’d need to change all the line changes to different place. Line changes can be left to the text editor and browser, which will automatically scale the text according to the screen size. The whole docs has been written without manual line changes.

Reply:
Done.